### DISCUSSION TITLE
Colocar um npc numa casa criado pelo player ?

### DISCUSSION THREAD

**Discussion by MauBrineGM** [ldtteam/minecolonies#8138] [Q&A]
Eu venho jogando a um tampo e recentimente comecei a usar os npc da colonia, mas eles só podem ter casas se eles criarem ,isso estraga minha villa ja q eu criei um terreno para cada um, e quando tento colocar um deles la dentro ele é obrigado a constroir , e a construção é enorme , tem como eu criar a casa e colocar ele dentro ? ,achei as casas dos npc muito feio ou muito enificas, tipo a casa do construtor que foi a primeira que  fiz  foi mó bonito com baus e varios infeites e quando ele crio a casa dele foi muito feio , se tiver como colocar um npc numa casa criado pelo Player me avisa

**Comment by uecasm** [comment]
It's best to ask support questions on Discord (although only in English -- use a translation site if needed).

Having said that, it's not possible to use player-created buildings (unless you go through the extra step of creating a custom schematic set -- ask about that on Discord if you like, but it's quite a bit of work).  The mod is designed such that only buildings built by the builder will work.  There are a large number of different building styles included in the mod (with a few extras only in modpacks), and you can mix and match them however you like, so there will hopefully be something that appeals to you.  They do also get fancier as you level them up.
